integrations/Posthog/browser.js

```diff
@@ -52,14 +59,16 @@ class Posthog {
       }, e.__SV = 1)
     }(document, window.posthog || []);
 
-    const configObject = {api_host: this.yourInstance,
+    const configObject = {
+      api_host: this.yourInstance,
       autocapture: this.autocapture,
       capture_pageview: this.capturePageView,
       disable_session_recording: this.disableSessionRecording,
       property_blacklist: this.propertyBlackList,
-      disable_cookie: this.disableCookie
+      disable_cookie: this.disableCookie,
+      persistence: "localStorage+cookie",
```

Comment:
Let's make this configurable. It should default to the previous behavior.

Reply:
We can follow this approach too. But I would suggest it keep `localStorage+cookie` as default in this way it should reduce the amount of data we send to `app.posthog.com` for each event by 2-4kb, improving deliverability. [PostHog/posthog.com#2239](https://github.com/PostHog/posthog.com/pull/2239)

Reply:
Agreed but we can do that if we can confirm that there is no negative impact to the existing users. Otherwise, I'd leave it to the user for making a conscious decision.

Reply:
Have made it configurable